wordpress-mobile/WordPress-Android
# WP Stories: further Tracks changes

Follow up from https://github.com/wordpress-mobile/WordPress-Android/pull/12074#issuecomment-653587714

Builds on top of https://github.com/wordpress-mobile/WordPress-Android/pull/12333

Changes in this PR:
- added tracks for `MEDIA_PICKER_OPEN_FOR_STORIES` so we track this start of flow unambiguously c059892
- fixed `editor_source` to also track the stories creator in fd5bf07
- also added property `has_wp_stories_blocks` for uploaded posts event `editor_post_published` in b6c4ff3

To test:
1. tap on the FAB, create a story
2. publish it
3. observe the logs for the correct tracks changes. Should look something like this (only pasting here the interesting parts):

```
2020-07-03 16:21:14.277 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_post_created, Properties: {"blog_id":179562158,"created_post_source_detail":"story-from-my-site","post_type":"post","post_format":"wpstory","created_post_source":"post-list","is_jetpack":true}
2020-07-03 16:21:14.284 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_session_start, Properties: {"has_unsupported_blocks":"0","editor":"wp_stories_creator","content_type":"new","startup_time_ms":0,"session_id":"3ac4b1da-df12-41d6-b686-b72d30ce792c","post_type":"post","blog_type":"jetpack","editor_has_hw_disabled":"0","unsupported_blocks":[]}


2020-07-03 16:21:41.961 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_post_publish_now_tapped, Properties: {"via":"prepublishing_nudges"}


2020-07-03 16:21:42.516 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: story_save_successful, Properties: {"blog_id":179562158,"frame_qty":1,"elapsed_time":476,"is_retry":false,"is_jetpack":true}
2020-07-03 16:21:42.567 21062-21831/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_photo_added, Properties: {"ext":"jpg","blog_id":179562158,"age_ms":562,"bytes":966748,"mime":"image\/jpeg","megapixels":4,"is_jetpack":true,"via":"device_library"}


2020-07-03 16:21:42.630 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: story_post_save_remotely, Properties: {"blog_id":179562158,"frame_qty":1,"is_jetpack":true}


2020-07-03 16:21:42.724 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_session_end, Properties: {"has_unsupported_blocks":"0","editor":"wp_stories_creator","content_type":"new","session_id":"3ac4b1da-df12-41d6-b686-b72d30ce792c","post_type":"post","blog_type":"jetpack","editor_has_hw_disabled":"0","outcome":"cancel"}


2020-07-03 16:21:50.341 21062-21062/org.wordpress.android.prealpha I/WordPress-STATS: 🔵 Tracked: editor_post_published, Properties: {"editor_source":"wp_stories_creator","word_count":1,"blog_id":179562158,"post_id":77,"has_gutenberg_blocks":true,"post_type":"post","post_format":"standard","is_jetpack":true,"has_wp_stories_blocks":true}

```

PR submission checklist:
